jujugui/static/gui/src/app/components/entity-details/content/test-content.js

```diff
@@ -782,8 +786,8 @@ describe('EntityContent', function() {
         plans={[]}
         pluralize={pluralize}
         renderMarkdown={renderMarkdown} />, true);
-    var output = renderer.getRenderOutput();
-    var expected = (undefined);
+    const output = renderer.getRenderOutput();
+    const expected = (undefined);
```

Comment:
Not really; rendering the markdown is the responsibility of the marked library, which gets stubbed out in all our unit tests. This gets into the grey area between unit and functional tests, but I agree it would be good to have something asserted on the rendering.